listen/array-changes.js

```diff
@@ -14,16 +14,15 @@
 
 require("../shim");
 var List = require("../list");
-var WeakMap = require("weak-map");
 var PropertyChanges = require("./property-changes");
 var RangeChanges = require("./range-changes");
 var MapChanges = require("./map-changes");
-
-var array_splice = Array.prototype.splice;
-var array_slice = Array.prototype.slice;
-var array_reverse = Array.prototype.reverse;
-var array_sort = Array.prototype.sort;
-var array_swap = Array.prototype.swap;
+var ArrayP = Array.prototype;
+var array_splice = ArrayP.splice;
+var array_slice = ArrayP.slice;
+var array_reverse = ArrayP.reverse;
+var array_sort = ArrayP.sort;
+var array_swap = ArrayP.swap;
```

Comment:
This style change detracts from readability and has negligible impact on performance.